integration/simulation/transaction_injector.go

```diff
@@ -94,6 +135,11 @@ func (m *TransactionInjector) Start() {
 		return nil
 	})
 
+	wg.Go(func() error {
+		m.issueAwaitedRandomDeposits()
```

Comment:
what is the difference between these and the normal  deposits?

Reply:
Refactored as discussed offline 👍